Assets/Drag.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.UI;
+using UnityEngine;
+
+public class Drag : MonoBehaviour
+{
+    ManipulationHandler manipulationHandler;
+    Vector3 startingPosition;
+    GameObject clone;
+
+    private void Awake()
+    {
+        //Debug.Log("running awake");
+        startingPosition = transform.position;
+        manipulationHandler = GetComponent<ManipulationHandler>();
+        manipulationHandler.OnManipulationStarted.AddListener(startedMotion);
+        manipulationHandler.OnManipulationEnded.AddListener(stoppedMotion);
+    }
+
+    private void stoppedMotion(ManipulationEventData arg0)
```

Comment:
Functions/methods in C# are camel case but actually start with a capital letter (I know, bizarre) but just look at `GetComponent<>()` so make sure to write them camel starting with capital